### PR TITLE
Remove custom browser border

### PR DESCRIPTION
**Pull request by NestorEncinas**
JIRA: ONE-4762

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1554/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3132/ : SUCCESS in 11m 36s

File: libs/sdk-ui-kit/styles/scss/main.scss [hunk 1]
```diff
@@ -14,3 +14,4 @@
 @import "./header";
 @import "./messages";
 @import "./overlay";
+@import "./remove";
```
Comment (ivan-nejezchleb) [inline libs/sdk-ui-kit/styles/scss/main.scss:17]: should not be this "reset" imported as first, to allow others to easily override this?

Comment (NestorEncinas) [reply libs/sdk-ui-kit/styles/scss/main.scss:17]: Yes, true. The import order, will move it to the top

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1572/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3151/ : SUCCESS in 10m 35s

**Review by ivan-nejezchleb — APPROVED** [review]

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/512/ : SUCCESS in 33s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/772/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3153/ : SUCCESS in 13m 47s

**Pull request merged**